gcloud-java-core/src/main/java/com/google/gcloud/BaseServiceException.java

```diff
@@ -48,10 +48,16 @@
 
     private final Integer code;
     private final String reason;
+    private final boolean rejected;
 
     public Error(Integer code, String reason) {
+      this(code, reason, false);
+    }
+
+    public Error(Integer code, String reason, boolean rejected) {
```

Comment:
Should we use this constructor also in other services when it applies? For instance also in storage we retry on `429—Too Many Requests`.